ResearchKit/Localized/en.lproj/Localizable.strings

```diff
@@ -83,6 +83,9 @@
 "RANGE_ALERT_MESSAGE_BELOW_MAXIMUM" = "%1$@ is less than the minimum allowed value (%2$@).";
 "RANGE_ALERT_MESSAGE_OTHER" = "%@ is not a valid value.";
 
+/* Alert for invalid email address value */
+"INVALID_EMAIL_ALERT_MESSAGE" = "Invalid email address: ";
```

Comment:
From localization standpoint, better to use a specifier like this: `"Invalid email address: %@"`. In case, some language does not put the typed in email address at the end.